cpp/src/gandiva/projector_cache_key.h

```diff
@@ -41,7 +41,7 @@ class ProjectorCacheKey {
       boost::hash_combine(result, expr_as_string);
       UpdateUniqifier(expr_as_string);
     }
-    boost::hash_combine(result, configuration->Hash());
+    // boost::hash_combine(result, configuration->Hash());
```

Comment:
I'm not sure that We want to hash the precompiled bitcode, so I've disabled it.

Reply:
maybe, keep this Hash(), but dummy out the configuration->Hash() (just return 0) ?